pkg/wrangler/context.go

```diff
@@ -188,6 +204,29 @@ func (w *Context) OnLeader(f func(ctx context.Context) error) {
 	w.leadership.OnLeader(f)
 }
 
+func (w *Context) checkGVK(ctx context.Context, gvk schema.GroupVersionKind) error {
+	logrus.Warnf("cache for '%s' did not sync", gvk.String())
+	crd, err := w.CRD.CustomResourceDefinition().Get(gvk.GroupKind().String(), metav1.GetOptions{})
+	if err != nil {
+		return fmt.Errorf("failed to get crd for gvk: %s", err)
+
+	}
+
+	var enabled bool
+	for _, v := range crd.Spec.Versions {
+		if v.Served {
+			enabled = true
+			break
```

Comment:
You could drop `enabled` and return `nil` here...

Reply:
You could also use https://pkg.go.dev/slices#ContainsFunc here

Reply:
Ope that's some rookie behavior on my part. Fixed